src/backend/distributed/transaction/coordinated_transaction_management.c

```diff
@@ -20,9 +20,10 @@
 #include "access/twophase.h"
 #include "access/xact.h"
 #include "distributed/connection_management.h"
+#include "distributed/coordinated_transaction_management.h"
+#include "distributed/distributed_transaction_management.h"
```

Comment:
I think we should just leave this in one header, I don't see what we gain by this split.